components/sandbox/index.js

```diff
@@ -5,19 +5,32 @@ import ResizableIframe from 'components/resizable-iframe';
 
 export default class Sandbox extends wp.element.Component {
 
+	// Two phase render for sandboxed content.
+	// 1) Write the html into the hidden iframe's document. This
+	//    makes sure all the external content gets loaded, so
+	//    we have size information for images, etc. This is needed
+	//    because the MutationObserver on the resizing iframe can
+	//    sometimes pick up that there's an images added to the DOM,
+	//    and triggers a resize *before* the image has loaded, and so
+	//    goes not get the size information, and so doesn't resize
+	//    correctly. The hidden iframe is sandboxed and will not
+	//    allow scripts to run, because there was a condition where
+	//    scripts were running while the onLoad triggered, and we
+	//    didn't get consistent information about assets.
+	// 2) When the document has loaded, shift the content into the
+	//    resizing iframe.
+	// 3) Create new script elements where needed.
+
```

Comment:
A couple of typos in this comment: "an images", "goes not get".

I don't follow this bit either:

> The hidden iframe is sandboxed and will not allow scripts to run, because there was a condition where scripts were running while the onLoad triggered, and we didn't get consistent information about assets.

This and point 3 don't seem to be part of the two-phase rendering setup, rather something additional related to handling scripts?  Maybe that should be moved to a separate part of the comment?